From 1bff3b8cd74a95e7c3918670fb8cffcc61e088c2 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Thu, 30 Aug 2018 17:45:47 +0800
Subject: [PATCH 4614/5242] MLK-19317-1 drm/bridge: sec-dsim: add missing
 'ctrail' assignment

commit  cfd20952d2712409982329ec6c1bce54486e154a from
https://source.codeaurora.org/external/imx/linux-imx.git

In the macro 'DSIM_DPHY_TIMING' definition, the field
'clk_trail' assignment to 'ctrail' is missing which
certainly needs to be added.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit f2818410d3d8d3b09002a85b593cee192d60bb06)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/drm/bridge/sec_mipi_dsim.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/drm/bridge/sec_mipi_dsim.h b/include/drm/bridge/sec_mipi_dsim.h
index 4484218..2b125b2 100644
--- a/include/drm/bridge/sec_mipi_dsim.h
+++ b/include/drm/bridge/sec_mipi_dsim.h
@@ -54,6 +54,7 @@ struct sec_mipi_dsim_dphy_timing {
 	.clk_prepare	= cpre,					\
 	.clk_zero	= czero,				\
 	.clk_post	= cpost,				\
+	.clk_trail	= ctrail,				\
 	.hs_prepare	= hpre,					\
 	.hs_zero	= hzero,				\
 	.hs_trail	= htrail,				\
-- 
1.7.9.5

